pkg/api/handlers/libpod/networks.go

```diff
@@ -31,8 +31,19 @@ func CreateNetwork(w http.ResponseWriter, r *http.Request) {
 		return
 	}
 
+	query := struct {
+		IgnoreIfExists bool `schema:"ignoreIfExists"`
+	}{
+		IgnoreIfExists: false,
```

Comment:
bools are false by default so you do not need to explicitly set this line.

Reply:
Right, fixed